libobs/obs.c

```diff
@@ -263,8 +263,13 @@ static int obs_init_graphics(struct obs_video_info *ovi)
 	bool success = true;
 	int errorcode;
 
+	#ifdef EVERCAST
```

Comment:
Is this the same flag as the vendor skins?

Reply:
given it's a root OBS file, how do we go about modifying this? I'm 99% sure this wouldn't be compatible with upstream ie. ifdef <company/vendor> ad infinitum